pkgs/tools/security/passff-host/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, python3, pass }:
+
+stdenv.mkDerivation rec {
+  name = "passff-host-${version}";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "passff";
+    repo = "passff-host";
+    rev = version;
+    sha256 = "1zks34rg9i8vphjrj1h80y5rijadx33z911qxa7pslf7ahmjqdv3";
+  };
+
+  buildInputs = [ python3 ];
+
+  patchPhase = ''
+    substituteInPlace src/passff.py \
+      --replace /usr/bin/pass ${pass}/bin/pass
+  '';
+
+  preBuild = "cd src";
+  postBuild = "cd ..";
+
+  installPhase = ''
+    install -D bin/testing/passff.py $out/passff.py
+    cp bin/testing/passff.json $out/passff.json
```

Comment:
I think this shouldn't go to $out/ directly, but maybe `$out/share/passff-host/passff.json`?

Reply:
Good point

Reply:
I fixed that